### PR TITLE
feat: create slide film and slide advertisement components

### PR DESCRIPTION
**Pull request by cuongdc03**
Show filmcard and advertisement by using Slide
86epkx6y5

### CODE REVIEW

**Review by ces-anhlam — APPROVED** [review]

File: src/assets/Grandma.jpg
[the remark this answers is not in the document]
Comment (TonNQ) [reply src/assets/Grandma.jpg:1]: fixed

File: src/apis/cinema.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { customFetch, showError } from '@/utils/http'
+import { CINEMA_URL } from './constant'
+
 export const getAllCinemas = async () => {
-    const respond = await customFetch.get('/cinema', { signalKey: 'getAllCinemas' }).catch(showError)
-    return respond.map(cinema => ({ id: cinema.id, name: cinema.name }));
-  }
-  
\ No newline at end of file
+  const respond = await customFetch.get(CINEMA_URL, { signalKey: 'getAllCinemas' }).catch(showError)
```
Comment (ces-phatnguyen) [inline src/apis/cinema.js:5]: typo => "response"

File: src/components/SlideAdvertisement/constant.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import drinkPopcornImage from '@/assets/bap-nuoc-onl.webp'
+import phim1Image from '@/assets/phim1.webp'
```
Comment (ces-phatnguyen) [inline src/components/SlideAdvertisement/constant.jsx:2]: English or Vietnamese, 1 language only

File: src/components/SlideFilm/SlideFilm.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        breakpoint: SMALL_SCREEN,
+        settings: {
+          slidesToShow: 1,
+          slidesToScroll: 1
+        }
+      }
+    ]
+  }
+
+  if (!films || films.length === 0) {
```
Comment (ces-phatnguyen) [inline src/components/SlideFilm/SlideFilm.jsx:45]: Should return before declaring settings since this code block does not use the variable setting

**Review by ces-phatnguyen — APPROVED** [review]

**Pull request merged**